_1327/main/migrations/0005_add_anonymous_group.py

```diff
@@ -4,13 +4,24 @@
 from django.db import migrations
 from guardian.management import create_anonymous_user
 from guardian.utils import get_anonymous_user
+from guardian.conf import settings as guardian_settings
+
+from _1327.user_management.models import UserProfile
+
 
 def add_anonymous_group(apps, schema_editor):
-	create_anonymous_user(None)
+	User = apps.get_model("user_management", "UserProfile")
+	user_arguments = {UserProfile.USERNAME_FIELD: guardian_settings.ANONYMOUS_USER_NAME}
+	anonymous_user = User(**user_arguments)
+	# anonymous_user.set_unusable_password()
+	anonymous_user.save()
```

Comment:
these are not "lint fixes", are they? :)

Reply:
Well, I just accidentally committed those changes. What should I do about them?